storybook/main.js

```diff
@@ -14,4 +14,12 @@ module.exports = {
 		babelModeV7: true,
 		emotionAlias: false,
 	},
+	webpackFinal: async ( config ) => {
```

Comment:
Same here

Reply:
Good point! I added a comment, Please have a look.

Reply:
Thanks for adding the comment! LGTM! 🚢